### ISSUE TITLE
Q3 - Q1 ≠ IQR from table generated by table1

### ISSUE THREAD

**Issue by vasili111** [benjaminrich/table1#116]
Unfortunately I cannot share data.

This is the code I am using:

```table1_object <- table1(tabble1_formula, data=ready_for_table1_df, render.continuous=c(.="Mean (SD)", .="Median [Min, Max]", .="IQR (Q3 - Q1)"))```

I am getting table generate:
![image](https://github.com/benjaminrich/table1/assets/7338799/f518a618-17c7-4eba-8b6a-cb40bfc4e2c9)

Interquartile range (IQR) that we got from software is `22.3`. But if we calculate IQR by `Q3 - Q1 = 65.0 - 42.8 = 22.2`. My assumption would be that software first subtracts non-rounded `Q3 - Q1` and after rounds the result and that may be why we see that the difference of the rounded `Q3 - Q1` is different from the IQR that is provided by the software.

I think it would be better if table1 library will provide IQR 22.2 in this case.

**Comment by benjaminrich** [comment]
I agree that it is a rounding issue, but I disagree with your conclusion that it would be better to do it the other way. The approach taken is to round late to keep more precision. If you want to do it the other way it's possible by customizing the render function.
